### PR TITLE
React: Contact details

### PR DESCRIPTION
**Pull request by Krijovnick**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
           { fullName: 'material.blue.dark.compact', name: 'material.dark' }
         ]
         ARGS: [
-          { project: 'angular', page: 'crm-contact-form' },
```
Comment (const314) [inline .github/workflows/test.yml:47]: These are the names of the test files from the testing package. Rename the 'crm-contact-form.test.js' file as well.

File: packages/react/src/components/card-opportunities/CardOpportunities.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <Button
+      text='Add Opportunity'
+      icon='add'
+      width={300}
+      height={115}
+      stylingMode='outlined'
+      type='default'
+      className='add-tile'
+      onClick={addOpportunity}
+    ></Button>
```
Comment (const314) [inline packages/react/src/components/card-opportunities/CardOpportunities.tsx:56]: Use self-closing tags for components without children

File: packages/react/src/components/card-tasks/CardTasks.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    ]);
+  }, [gridData]);
+
+  return (
+    <DataGrid
+      dataSource={gridData}
+      columnAutoWidth
+      selectionFilter={selectionFilter}
+    >
+      <Selection mode='multiple' deferred></Selection>
```
Comment (const314) [inline packages/react/src/components/card-tasks/CardTasks.tsx:37]: Use self-closing tags for components without children

File: packages/react/src/components/status-task/StatusTask.tsx [hunk 4]
```diff
@@ -1,10 +1,8 @@
 import React from 'react';
 import { TaskProp } from '../../shared/types/task';
 
-import './StatusTask.scss';
-
-export const StatusTask = ({ text }: TaskProp) => (
+export const StatusTask = ({ text, showText = true }: TaskProp) => (
```
Comment (const314) [inline packages/react/src/components/status-task/StatusTask.tsx:4]: The StatusTask, PriorityTask, and ContactStatus components look very similar. Maybe extract some generic component?

File: packages/react/src/components/priority-task/PriorityTask.tsx [hunk 5]
```diff
@@ -3,9 +3,9 @@ import { TaskProp } from '../../shared/types/task';
 
 import './PriorityTask.scss';
 
-export const PriorityTask = ({ text }: TaskProp) => (
+export const PriorityTask = ({ text, showText = true }: TaskProp) => (
   <div className={`priority-task priority-${text.toLowerCase()}`}>
     <div className='separator-task'></div>
-    <span>{text}</span>
+    <span>{showText ? text : false}</span>
```
[the remark this answers is not in the document]
Comment (Krijovnick) [reply packages/react/src/components/priority-task/PriorityTask.tsx]: this is typo. fixed

File: packages/react/src/components/task-form/TaskFormDetails.tsx [hunk 6]
```diff
@@ -18,146 +19,134 @@ import { PRIORITY_ITEMS, STATUS_ITEMS } from '../../shared/constants';
...
+  </div>
 );
 
 const renderStatusField = (data: string) => (
-  <>
-    <StatusTask text={data}></StatusTask>
-    <TextBox readOnly></TextBox>
-  </>
+  <div className='status-task-editor-field'>
+    <StatusTask text={data} showText={false}></StatusTask>
```
Comment (dxvladislavvolkov) [inline packages/react/src/components/task-form/TaskFormDetails.tsx:37]: Do we need to use status task here? Looks like we can use only text-box

Comment (Krijovnick) [reply packages/react/src/components/task-form/TaskFormDetails.tsx]: status task removed

**Review by const314 — DISMISSED** [review]

**Review by MikeVitik — DISMISSED** [review]

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**